.circleci/config.yml

```diff
@@ -51,47 +54,59 @@ jobs:
       - checkout
       - run: apt update && apt install ragel
```

Comment:
glib and rest of packages installed on travisci should be installed here for tests I guess.